zds/forum/models.py

```diff
@@ -599,10 +603,10 @@ def is_read(topic, user=None):
     :param user: A user. If undefined, the current user is used.
     :return:
     """
-    if user is None:
-        user = get_current_user()
-
-    return TopicRead.objects.filter(post=topic.last_message, topic=topic, user=user).exists()
+    return TopicRead \
+        .objects \
+        .filter(post=topic.last_message, topic=topic, user=user or get_current_user()) \
```

Comment:
Sur plusieurs lignes ?

Reply:
Pas sûr que ça améliore la lisibilité. ZdS a des conventions précises autre que PEP8 ?

Autre moyen, à part revenir sur le `if not user` avant :

```python
user = user or get_current_user()

return TopicRead \
    .objects \
    .filter(post=topic.last_message, topic=topic, user=user) \
    .exists()
```

D'ailleurs y'a pas mal d'autres endroits où 

```python
if not user:
    user = get_current_data()
```

pourrait être remplacé par ça si la plupart sont d'accord.

Reply:
Je disais ça car [ici](https://github.com/zestedesavoir/zds-site/pull/4621/files#diff-d9e47ece3e055339353c712faa8ad7edR363) tu mets les arguments de `filter()` sur plusieurs lignes :) Mais ta solution alternative me convient !